lib/cipherstash/analysis/text_processor.rb

```diff
@@ -46,8 +46,19 @@ def build_token_filters(array)
             TokenFilters::Downcase.new(obj)
 
           when "ngram"
-            TokenFilters::NGram.new(obj)
+            if obj["tokenLength"]
+              raise CipherStash::Client::Error::InternalError, "'tokenLength' is deprecated. Use 'minLength' and 'maxLength' for the ngram filter."
```

Comment:
`InternalError` might be a bit misleading here. Isn't this a configuration error on the user's part rather than an internal error within ruby-client?

Nit: if we're raising an error here the feature is really no longer supported rather than deprecated (as the error message suggests).

Reply:
yeah good call, i'll update this to be a client error